language/move-prover/tests/sources/functional/fixed_point_arithm.move

```diff
@@ -108,7 +108,7 @@ module 0x42::FixedPointArithmetic {
     }
     spec mul_div_incorrect {
         ensures result >= x; // disproved
-        ensures result == x; // disproved
+        //ensures result == x; // disproved; commented out because of timeout
```

Comment:
- add a TODO
- and how often is the timeout happening (and if we increase the timeout, can this be proved?)

Reply:
This is an interesting case. If you prove these four ensures separately, all of them will be disproved with counter examples. But the exp file only has the error info for ">=" and "<". That is why I commented out the case with "==".